packages/lib/server/username.ts

```diff
@@ -106,7 +109,9 @@ const usernameHandler =
     return handler(req, res);
   };
 
-const usernameCheck = async (usernameRaw: string) => {
+const usernameCheck = async (usernameRaw: string, currentOrgDomain?: string | null) => {
```

Comment:
If we're going to use this function instead of `checkPremiumUsername` can we add some unit tests?

Reply:
Haven't started using it as a replacement of checkPremiumUsername. That would be a followup thing. Would certainly need some unit tests when that happens